feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -165,6 +165,7 @@ func fetchPackets(ctx context.Context, t *testing.T, client *p4rt_client.P4RTCli
 			t.Fatalf("There is error seen when receving packets. %v, %s", err, err)
 			break
 		}
+	time.Sleep(1 * time.Second)
```

Comment:
Why is the sleep needed? I don't see anything timing sensitive in the test loop, so I'm guessing this is to workaround a bug in the p4rt_client or the server? Please explain in a code comment.

Reply:
yes. it is a workaround of b/268530188.

Reply:
As this is opensourced code, we should provide a public reference and not an internal bug.  Since this explanation is short, please include comments in the file.

Reply:
I just talked to @Ankur19, and he confirmed that this is an issue in the p4rt_client that he will contribute a fix next sprint.